src/static/model/retros/manager.js

```diff
@@ -0,0 +1,67 @@
+// todo: When he have persistance, populate this pool
+var retros = {};
+
+function Retro(name, ownerUsername, participants) {
+    this.name = name;
+    this.owner = ownerUsername;
+    this.participants = participants || [];
+    this.isActive = true;
```

Comment:
Maybe we can have some kind of status, and use isActive as a method.
Cause will also need to know on what step of the retro we are (i.e.: adding items, voting, adding action items, closed)